Input/Core/InputEventPool.cs

```diff
@@ -1,26 +1,45 @@
 using System;
 using System.Collections.Generic;
-using UnityEngine;
 
 namespace UnityEngine.InputNew
 {
 	class InputEventPool
 	{
-		#region Public Methods
+		readonly Dictionary<Type, List<InputEvent>> m_Pools = new Dictionary<Type, List<InputEvent>>();
 
+		#region Public Methods
```

Comment:
non-uniform use of regions sometimes have a new line between inner-region code, and other times not (P.S. I know this isn't your code, just saying we should unify conventions). Personally I prefer the use of regions, and am glad to see them being used ;-).

Reply:
I'm against regions, and our conventions explicitly disallow them, AFAIK. The only time I like a region is if you are intentionally trying to hide boilerplate method code. There is a lot of discussion on this
https://softwareengineering.stackexchange.com/questions/53086/are-regions-an-antipattern-or-code-smell